src/AppBundle/Referent/ReferentDatabaseDumper.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace AppBundle\Referent;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Exception\ReferentNotFoundException;
+use AppBundle\Repository\AdherentRepository;
+use League\Flysystem\FilesystemInterface;
+
+class ReferentDatabaseDumper
+{
+    const STORAGE_SPACE = 'dumped_referents_users';
+
+    const EXPORT_TYPES = [
+        'all',
```

Comment:
I thought about it but it doesn't really make sense so far because we don't really hardcode the names outside of the class itself. Only in the unit tests of course. This tool is called from a console command where the type argument must be given to the command.